extensions/Cake.SquareLogo.yml

```diff
@@ -0,0 +1,16 @@
+Type: Addin
+Name: Cake.SquareLogo
+NuGet: Cake.SquareLogo
+Assemblies:
+- /**/Cake.SquareLogo.dll
+Repository: 
+ProjectUrl: https://www.nuget.org/packages/Cake.SquareLogo/
+Author: Cake.SquareLogo
+Description: Package Description
```

Comment:
Do we want to improve metadata (description, categories) manually?

Reply:
I suggest we let AddinDiscoverer create the YAML file with the description and tags from the package nuspec and manually overwrite after the PR is merged. For that point on, the discoverer will respect the manually edited description and categories and will not overwrite them.